pallets/xyk/src/lib.rs

```diff
@@ -150,6 +153,7 @@ pub mod pallet {
 		BuyAssetAmountInvalid, // no tests
```

Comment:
still true?

Reply:
this is bit tricky to come up with scenario because this error is returns when buy price + fee overflows - so i cant think of way to generate such scenario.